src/Schema/Comparator.php

```diff
@@ -432,95 +434,76 @@ public function diffForeignKey(ForeignKeyConstraint $key1, ForeignKeyConstraint
     /**
      * Returns the difference between the columns
      *
-     * If there are differences this method returns $field2, otherwise the
-     * boolean false.
+     * If there are differences this method returns the changed properties as a
+     * string array, otherwise an empty array gets returned.
```

Comment:
Although this looks right, it's a breaking change which isn't acceptable in a minor release.

Reply:
The old method description was wrong, there is no `$field2` and it never returned `false`. I fixed the description, the behavior is the same as before.